vendor/engines/secure_rooms/app/presenters/secure_rooms/link_collection_extension.rb

```diff
@@ -0,0 +1,23 @@
+module SecureRooms
+
+  module LinkCollectionExtension
+
+    extend ActiveSupport::Concern
+
+    included do
+      insert_index = tab_methods.index(:admin_products) || -1
+      tab_methods.insert(insert_index, :admin_occupancies)
+    end
+
+    def admin_occupancies
+      if single_facility? && SecureRoom.for_facility(facility).exists?
```

Comment:
for some reason we wrote the story to show the tab "archived or not" but I don't remember why
https://www.pivotaltracker.com/story/show/144059001

do we need to update story?